packages/components/pie-tag/README.md

```diff
@@ -74,16 +74,40 @@ import { PieTag } from '@justeattakeaway/pie-tag/dist/react';
 
 | Property | Type | Default | Description |
 |---|---|---|---|
-| - | - | - | - |
+| size | `String` | `large` | Size of the tag. Can be either `large` or `small` |
+| variant | `String` | `neutral` | Variant of the tag, one of `variants` - `neutral-alternative`, `neutral`, `outline`, `ghost`, `blue`, `green`, `yellow`, `red`, `brand` |
+| isStrong | `Boolean` | `false` | If `true`, displays strong tag styles for `green`, `yellow`, `red`, `blue` and `neutral` variants'|
+
+Since the component is not interactive it doesn't have a disabled property. To give the Tag a disabled look please use `--tag-opacity` css variable. Recommended opacity level for disabled tag is 0.5.
```

Comment:
wouldn't this approach get applied to all tags on the page since the variable isn't unique? for example, i have a Tag used in a header, and then another one in the footer, setting the variable to 0.5 will make both tags dimmed? if not, I feel this might be an extra load to our end users, maybe we can introduce a prop called isDimmed or something instead of isDisabled?

Reply:
It shouldn't. For example if you have a disabled card on the page and you want a tag inside the card to look disabled you can override `--tag-opacity` only for a tag inside a disabled card smth like `.card[disabled] .tag {}`. Does this make sense?

Reply:
yea that makes a lot of sense, tho, I think having the behaviour applied as a prop might reduce the overhead of end users and will provide a simpler API imo.